src/wasm/kotlin/java/arcs/wasm/WasmRuntimeClient.kt

```diff
@@ -0,0 +1,55 @@
+package arcs
+
+import arcs.wasm.*
+
+actual object RuntimeClient {
+    actual fun <T : Entity<T>> singletonClear(particle: Particle, singleton: Singleton<T>) =
+        singletonClear(particle.toAddress(), singleton.toAddress())
+
+    actual fun <T : Entity<T>> singletonSet(particle: Particle, singleton: Singleton<T>, encoded: String) =
+        singletonSet(particle.toAddress(), singleton.toAddress(),
+            encoded.toWasmString())
+
+    actual fun <T : Entity<T>> collectionRemove(particle: Particle, collection: Collection<T>, encoded: String) =
+        collectionRemove(particle.toAddress(), collection.toAddress(),
+            encoded.toWasmString())
+
+    actual fun <T : Entity<T>> collectionClear(particle: Particle, collection: Collection<T>) =
+        collectionClear(particle.toAddress(), collection.toAddress())
+
+    actual fun <T : Entity<T>> collectionStore(
+        particle: Particle,
+        collection: Collection<T>,
+        encoded: String): String? {
+        val wasmId = collectionStore(particle.toAddress(), collection.toAddress(), encoded.toWasmString())
+        return wasmId.toNullableKString()?.let { _free(wasmId); it }
+
+    }
+
+    actual fun log(msg: String) = arcs.wasm.log(msg);
+
+    actual fun onRenderOutput(particle: Particle, template: String?, model: String?) =
+        onRenderOutput(particle.toAddress(), template.toWasmNullableString(),
+            model.toWasmNullableString())
+
+    actual fun serviceRequest(particle: Particle, call: String, encoded: String, tag: String) =
+        serviceRequest(particle.toAddress(), call.toWasmString(),
+            encoded.toWasmString(), tag.toWasmString())
```

Comment:
When wrapping params (here and elsewhere in the PR), it's preferred to use this style:

```kotlin
serviceRequest(
    particle.toAddress(),
    call.toWasmString(),
    encoded.toWasmString(),
    tag.toWasmString()
)
```